gallery/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sonatype/react-shared-components-gallery",
-  "version": "8.7.0",
+  "version": "8.8.0",
```

Comment:
Should definitely be a patch version

Reply:
> Should definitely be a patch version

Updated to correct version.
Check b1871f57592f88bbf1d87580c996934d74a97910